### ISSUE TITLE
NVRTC_ERROR_NO_LOWERED_NAMES_BEFORE_COMPILATION

### ISSUE THREAD

**Issue by UselessGuru** [sp-hash/TeamBlackMiner#387]
`TeamBlackMiner-v2.01\TBMiner.exe --algo kawpow --hostname eu.mining.prohashing.com --wallet ********************* --port 3361 --server-passwd a=kawpow,e=off,n=***************,o=********************** --api --api-version 1.4 --api-port 4007 --cuda-devices [0]`

```
00:00:00 [2023-09-30 19:10:02.702]
00:00:00 [2023-09-30 19:10:02.766] OpenCL driver version: 536.99
00:00:00 [2023-09-30 19:10:02.798] Cuda driver version: 12.2
00:00:00 [2023-09-30 19:10:02.798] Cuda runtime API version: 12.0
00:00:00 [2023-09-30 19:10:02.798]
00:00:00 [2023-09-30 19:10:02.798]
00:00:00 [2023-09-30 19:10:02.798]
00:00:00 [2023-09-30 19:10:02.798] Welcome to Team Black Miner 2.01
00:00:00 [2023-09-30 19:10:02.798]
00:00:00 [2023-09-30 19:10:02.798]
00:00:00 [2023-09-30 19:10:03.099] GPU0 NVIDIA GeForce GTX 1660 SUPER 6GB added to miner as Cuda device
00:00:00 [2023-09-30 19:10:03.099] Trying eu.mining.prohashing.com:3361 (kawpow)
00:00:00 [2023-09-30 19:10:03.099] Api server started at 127.0.0.1:4007
00:00:00 [2023-09-30 19:10:03.099] GPU0 XIntensity set to 1024
00:00:00 [2023-09-30 19:10:03.162] eu.mining.prohashing.com: Pool responded to subscribe
00:00:00 [2023-09-30 19:10:03.178] GPU0 Starting generating kawpow dag (Epoch: 399)
00:00:00 [2023-09-30 19:10:03.257] eu.mining.prohashing.com: Pool responded successfully to authorize
00:00:00 [2023-09-30 19:10:05.904] GPU0 DAG generated in 2.73 seconds (4317178kb)
00:00:00 [2023-09-30 19:10:05.907] Job a0c7381b
00:00:00 [2023-09-30 19:10:05.907] C:\Users\frode\Downloads\tbminer\source\sp_kawpow\cuda_kawpow.cpp:cuda_kawpow_compile_kernel():547: A runtime error occured: NVRTC_ERROR_NO_LOWERED_NAMES_BEFORE_COMPILATION
00:00:00 [2023-09-30 19:10:05.907] Shutting down threads
00:00:00 [2023-09-30 19:10:05.923] Control thread exited
00:00:00 [2023-09-30 19:10:05.923] Stratum thread exited
00:00:00 [2023-09-30 19:10:05.923] GPU0 thread exited
```

Same error also with 1.99 & 2.00.
What is missing?

**Comment by UselessGuru** [comment]
Any news?

**Comment by sp-hash** [comment]
fixed in v2.02
